### PR TITLE
Updated sections in the documentation related to the installer

### PR DESCRIPTION
**Pull request by yuchaoran2011**
### What changes were proposed in this pull request?
Tell the user about how to use the new installer.

### Why are the changes needed?
The old installer is deprecated. The new one wasn't properly documented.

### Does this PR introduce any user-facing change?
Yes. Updated user-facing docs.

### How was this patch tested?
By looking at the adoc rendering.

### CODE REVIEW

File: docs/docs-source/docs/modules/ROOT/nav.adoc [hunk 1]
```diff
@@ -12,7 +12,7 @@ include::partial$include.adoc[]
 *** xref:get-started:develop-example-streamlets.adoc[Develop example streamlets]
 *** xref:get-started:create-example-blueprint.adoc[Create the example blueprint]
 ** xref:get-started:run-in-sandbox.adoc[Run in a local sandbox]
-** xref:get-started:setup-gke-cluster.adoc[Set up a GKE cluster]
+** xref:get-started:setup-k8s-cluster.adoc[Set up a Kubernetes cluster]
```
Comment (olofwalker) [inline docs/docs-source/docs/modules/ROOT/nav.adoc:15]: Why is this changed from `GKE` to `Kubernetes`? The rest of the text refers to GKE.

File: docs/docs-source/docs/modules/get-started/pages/setup-k8s-cluster.adoc [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+:toc-title: ON THIS PAGE
+:toclevels: 2
+
+include::ROOT:partial$include.adoc[]
+
+This page walks you through the important steps to get Cloudflow up and running. For more detailed instructions on how to use or build your own Cloudflow installer, refer to the https://github.com/lightbend/cloudflow/tree/master/installer[installer section] of the GitHub repo.
+
+To create a new cluster, clone the https://github.com/lightbend/cloudflow/[repo] in your local system by selecting the latest release branch or download the latest release file from https://github.com/lightbend/cloudflow/releases[here], go to the `installer/` folder and follow the instructions below.
+
+* Run `gcloud init` if you are creating a GKE cluster and this is your first time doing so. This command initializes the gcloud system on your local and performs necessary settings of gcloud credentials. You can have a look at the details in the https://cloud.google.com/sdk/gcloud/reference/init[GKE documentation].
```
Comment (maasg) [inline docs/docs-source/docs/modules/get-started/pages/setup-k8s-cluster.adoc:12]: here, we give instructions for `gcloud` but a bit later, we offer the option to create a GKE, AKS, or, EKS cluster.
To be consistent, we either add login sections for each of them or we just refer to 'login to your cluster' and maybe add links to the (external) gke, ask, and, eks login instruction pages.

File: docs/docs-source/docs/modules/get-started/pages/setup-k8s-cluster.adoc [hunk 3]
```diff
@@ -0,0 +1,77 @@
...
+
+NOTE:: This can be done once you have a valid Google account and have access to a project set up for you by the administrator.
+
+* Create a Kubernetes cluster with a specific name on a cloud provider of your choosing. We provide cluster launching scripts for GKE, AKS and EKS. `./create-cluster-[gke|aks|eks].sh <cluster-name>`
+* Make sure your cluster has at least appropriate storage class that offers ReadWriteMany (RWM) capability. Most cloud providers have their preferred storage class for that purpose (e.g. AzureFile on AKS and EFS CSI on EKS). We also provide a script `./install_nfs.sh` that you can run to install NFS that will satisfy the RWM characteristic.
+* Go to the Github release page for your chosen Cloudflow version and download the bootstrap script. It's in the list of assets. For example, this is the https://github.com/lightbend/cloudflow/releases/tag/v1.3.3[page] for Cloudflow 1.3.3.
+* Run the bootstrap script to install Cloudflow.
+
```
Comment (maasg) [inline docs/docs-source/docs/modules/get-started/pages/setup-k8s-cluster.adoc:20]: add an example of running the script (for illustration purposes)
```
$ ./bootstrap-install-script-<CLOUDFLOW_VERSION>.sh 
```

File: docs/docs-source/docs/modules/get-started/pages/setup-k8s-cluster.adoc [hunk 4]
```diff
@@ -0,0 +1,77 @@
...
+This will take you through the process of installing Cloudflow in the cluster. In course of this it will ask a few questions regarding the storage class to be installed. The following should be the response to these:
```
Comment (maasg) [inline docs/docs-source/docs/modules/get-started/pages/setup-k8s-cluster.adoc:21]:
```suggestion
This will take you through the process of installing Cloudflow in the cluster. In the course of its execution, it will ask a few questions regarding the storage class to be installed. 
The following provides guidance on how to respond to these questions:
```

File: docs/docs-source/docs/modules/get-started/pages/setup-k8s-cluster.adoc [hunk 5]
```diff
@@ -0,0 +1,77 @@
...
++------------------------------------------------------------------------------------+
+
+NAME                                                         READY   STATUS              RESTARTS   AGE
+cloudflow-flink-operator-8588dbd8f4-gsjnm                    1/1     Running             0          70s
+cloudflow-operator-57f47676f7-svbvj                          0/1     ContainerCreating   0          3s
+cloudflow-sparkoperator-69669fdd54-88brw                     0/1     ContainerCreating   0          35s
+strimzi-cluster-operator-7ff64d4b7-c9r2n                     1/1     Running             0          51s
+```
+
+The complete installation takes a bit of a time, and you can follow the progress using `kubectl` command. All operators are installed in the namespace `cloudflow`. When everything is installed, you will see the following:
```
Comment (maasg) [inline docs/docs-source/docs/modules/get-started/pages/setup-k8s-cluster.adoc:55]:
```suggestion
The complete installation takes a few minutes. 
All operators are installed in the namespace `cloudflow`. 
You can follow the progress of the installation using the following `kubectl` command:
``bash
$ kubectl get pods -n cloudflow
``
When everything is installed, you will see the following output:
```

Comment (maasg) [reply docs/docs-source/docs/modules/get-started/pages/setup-k8s-cluster.adoc:55]: the suggestion syntax prevents me from using the triple quotes in the snippet. I made them double-quotes for the example but you'll need to update it before merging.

File: docs/docs-source/docs/modules/ROOT/nav.adoc (hunk 1 above)
Comment (yuchaoran2011) [reply docs/docs-source/docs/modules/ROOT/nav.adoc:15]: Because the instructions in the original `setup-gke-cluster.adoc` file are mostly generic that applies to other K8s flavors too, except for a few lines that are GKE specific. And the installer is also flavor-agnostic.

File: docs/docs-source/docs/modules/get-started/pages/setup-k8s-cluster.adoc (hunk 5 above)
Comment (yuchaoran2011) [reply docs/docs-source/docs/modules/get-started/pages/setup-k8s-cluster.adoc]: Fixed.
